libraries/vendor/composer/autoload_classmap.php

```diff
@@ -6,10 +6,772 @@
 $baseDir = dirname(dirname($vendorDir));
 
 return array(
+    'Cache_Lite' => $vendorDir . '/pear/cache_lite/Cache/Lite.php',
```

Comment:
We committing dev resources to repo here?  If not, run `composer install --no-dev` to dump the dev deps.